script.js

```diff
@@ -0,0 +1,9 @@
+// show/hide bio description
+function toggleDescription(button) {
+  button.target.parentElement.classList.toggle('bio-box__info--active');
+}
+
+const readMoreBtns = document.querySelectorAll('.bio-box__button');
```

Comment:
Yep, I originally wanted to use the 'onclick' property because it looks a bit cleaner but I saw that people don't really recommend using that anymore, so nice one 👍